views/components/feedbackBanner/feedbackBanner.njk

```diff
@@ -0,0 +1,9 @@
+<div class="govuk-!-display-none-print">
+  <div class="govuk-width-container">
+    <div class="feedback-banner" data-qa="feedback-banner">
+      <a href="https://eu.surveymonkey.com/r/GYB8Y9Q?source={{ currentUrlPath }}" class="govuk-link govuk-link--inverse" target="_blank" rel="noopener noreferrer">
```

Comment:
you need the host name in here - this is how the survey monkey questionnaire works out which service the feedback is for.

Worth having a look to see how this is done in PSH:
e.g: https://github.com/ministryofjustice/prisonstaffhub/blob/7c56f92fabfe712c4c599fb9e79eadd665b34deb/views/components/feedbackBanner/feedbackBanner.njk#L2

Reply:
Aha - thanks I missed this but all makes sense now, that bit had confused me a little.